sc-memory/sc-core/sc-store/sc_storage.c

```diff
@@ -1034,19 +1034,12 @@ sc_result sc_storage_get_link_content(const sc_memory_context *ctx, sc_addr addr
   {
     sc_uint8 len = el->content.data[0];
 
-    if (len != 0)
-    {
-      g_assert(len < SC_CHECKSUM_LEN);
-      gchar *buff = g_new0(gchar, len);
-      memcpy(buff, &el->content.data[1], len);
-      *stream = sc_stream_memory_new(buff, len, SC_STREAM_FLAG_READ, SC_TRUE);
+    g_assert(len < SC_CHECKSUM_LEN);
```

Comment:
Old code is stable. This one not.
Also empty sc_link should not generate sc_stream